### PR TITLE
django and pandas don't need changes

### PR DESCRIPTION
**Pull request by JelleZijlstra**

### CODE REVIEW

File: src/black_primer/primer.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     },
     "pandas": {
       "cli_arguments": ["--experimental-string-processing"],
-      "expect_formatting_changes": true,
+      "expect_formatting_changes": false,
```
Comment (JelleZijlstra) [inline src/black_primer/primer.json:98]:
```suggestion
      "expect_formatting_changes": true,
```
And now it needs changes again? Compare https://github.com/psf/black/runs/4446024299?check_suite_focus=true

**Review by cooperlees — APPROVED** [review]
Wow. Crazy of me not expecting changes here - I don't think django uses black.

**Comment by ichard26** [comment]
Seems like this isn't the case anymore? Not sure what happened here to be honest.

**Comment by JelleZijlstra** [comment]
Yeah, I guess they must have both been changed.

**Pull request closed without merging**